syntaxes/vscode-objectscript-output.tmLanguage.json

```diff
@@ -23,6 +23,14 @@
       "match": "^Compilation finished successfully in \\S+",
       "name": "markup.bold"
     },
+    {
```

Comment:
You can remove these changes now that this syntax is no longer used. Otherwise, looks good!